zipline-gradle-plugin/src/test/kotlin/app/cash/zipline/gradle/testFixturesJvm.kt

```diff
@@ -0,0 +1,53 @@
+package app.cash.zipline.gradle
+
+import app.cash.zipline.QuickJs
+import app.cash.zipline.loader.CURRENT_ZIPLINE_VERSION
+import app.cash.zipline.loader.ZiplineFile
+import app.cash.zipline.loader.ZiplineManifest
+import app.cash.zipline.loader.ZiplineModule
+import okio.Buffer
+import okio.ByteString
+import okio.ByteString.Companion.toByteString
+
+val alphaJs = """
```

Comment:
Would it be worthwhile for us to have a class `TextFixtures` that takes a `QuickJs` in its constructor?